widgetssdk/src/main/java/com/glia/widgets/auth/SiteApiKey.java

```diff
@@ -0,0 +1,9 @@
+package com.glia.widgets.auth;
+
+import androidx.annotation.NonNull;
+
+public class SiteApiKey extends com.glia.androidsdk.SiteApiKey {
+    public SiteApiKey(@NonNull String siteApiKeyId, @NonNull String siteApiKeySecret) {
```

Comment:
It would be great to add JavaDoc for this class.

Reply:
Actually, I prefer to merge these changes to avoid adding "proxy" classes.
Integrators will be able to use `import com.glia.androidsdk.SiteApiKey` and this class will be unuseful.

Reply:
@deniss-mc Andrii’s suggestion sounds logical. What do you think?

Reply:
@AndriiHorishniiMOC integrators won't be able to use `import com.glia.androidsdk.SiteApiKey` unless they import core sdk to the project
<img width="769" alt="Screenshot 2022-01-03 at 10 23 27" src="https://user-images.githubusercontent.com/49229744/147910956-6d9df511-94bb-416a-9699-6a7169877240.png">

Reply:
@deniss-mc yeh, it because used `implementation` keyword for `com.glia:android-sdk` in the `widgetssdk/build.gradle`. Integrators will be able to use import `com.glia.androidsdk.SiteApiKey` with this changes https://github.com/salemove/android-sdk-widgets/pull/148/files.

Reply:
<img width="769" alt="Screenshot 2022-01-04 at 15 00 36" src="https://user-images.githubusercontent.com/49229744/148062900-f0798a93-da16-43a1-a00b-7f707bec09c5.png">

this is in `develop` already  - added all changes - image taken from current state

Reply:
@deniss-mc , I've checked out this branch - MUIC-638. Cleaned up the project, update Gradle, and removed duplicated classes. Looks like it works, could you please verify this commit - d19922bde467532d6ef975f0e6ef2189d0b6c8f6

Reply:
Seems to work, didn't recognize the classes at first tho 👍  thanks!